src/api/forms/service.js

```diff
@@ -102,6 +122,7 @@ function formTitleToSlug(title) {
 }
 
 /**
+ * @typedef {import('@defra/forms-model').FormDefinition} FormDefinition
  * @typedef {import('./errors.js').FormAlreadyExistsError} FormAlreadyExistsError
```

Comment:
Tiny one, we've already exported `FormDefinition` from `../types.js`

Feel free to change that to `~/src/api/types.js` as we've not stuck to that pattern in `@jsdoc` much

```suggestion
 * @typedef {import('~/src/api/forms/errors.js').FormAlreadyExistsError} FormAlreadyExistsError
 * @typedef {import('~/src/api/types.js').FormDefinition} FormDefinition
```